iggtools/common/samples.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
```

Comment:
This is great.   In future perhaps we can group together the MIDAS shared analysis code in analysis/midas.py with the view of separating code paths that are used for analysis from those used for DB construction --- because analysis paths would likely run outside of AWS at some point.